src/app/editor/viewer/lib/sketch.service.ts

```diff
@@ -41,8 +45,16 @@ export class SketchService {
   }
 
   async process(file: File) {
-    this._data = await this.sketch2Json(file);
-    this.parseColors(this._data.pages);
+    try {
+      this._data = await this.sketch2Json(file);
+      this.parseColors(this._data.pages);
+    } catch (e) {
+      this.store.dispatch(
+        new InformUser(
+          'The design was created using an unsupported version so the result may not be accurate.'
```

Comment:
Are we sure that this exception is related to an unsupported version of sketchApp? It may be due to other errors in the 2 functions in the try block.

Reply:
Also, can we actually inform the users about the exact version they used vs the versions that xlayers support?

Reply:
Nvm, I'll merge this PR for now. We will open another one that will add the version checking.